zerver/openapi/zulip.yaml

```diff
@@ -9014,12 +9021,19 @@ paths:
                           in this organization.  Null if no default has been set.
                       realm_message_content_delete_limit_seconds:
                         type: integer
+                        nullable: true
                         description: |
                           Present if `realm` is present in `fetch_event_types`.
 
                           Messages sent more than this many seconds ago cannot be deleted
                           with this organization's
                           [message deletion policy](/help/configure-message-editing-and-deletion).
+
+                          'null' value means that there is not limit and the message can be deleted
```

Comment:
"A `null` value" would be better phrasing.

Reply:
Done